### DISCUSSION TITLE
.exportCSV options

### DISCUSSION THREAD

**Discussion by Ironmunkii** [primefaces/community#574] [PrimeNG] [answered]
When I run 
`.exportCSV()`
you can pass options as a param. 

What are the available options? In particular, I need to export to CSV as a certain unicode. I want to know if there's an option for that and what it is?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,
 Thanks for the feedback, we've added a type for the method now. It's as below

```
/**
 * Custom CSV export options.
 */
export interface ExportCSVOptions {
    /**
     * Boolean value determining whether to export all selected values.
     */
    selectionOnly?: boolean;
    /**
     * Boolean value determining whether to export all values.
     */
    allValues?: boolean;
}
```
